packages/react-kit/src/components/Popover/Popover.tsx

```diff
@@ -195,7 +195,7 @@ class RawPopover extends React.Component<TFullPopoverProps, TPopoverState> {
 		}
 
 		return (
-			<EventListener onResize={this.onResize} onScroll={this.onScroll} target="window">
+			<EventListener onResize={this.onResize} onScroll={this.onScroll} target={window}>
```

Comment:
This breaks server-side rendering. Please consider adding some checks like `typeof window !== 'undefined'` or similar